akka-cluster/src/main/scala/akka/cluster/ClusterHeartbeat.scala

```diff
@@ -57,12 +67,15 @@ private[cluster] object ClusterHeartbeatSender {
   /**
    * Sent at regular intervals for failure detection.
    */
-  final case class Heartbeat(from: Address) extends ClusterMessage with HeartbeatMessage with DeadLetterSuppression
+  final case class Heartbeat(from: Address, sequenceNr: Long, sendNanoTime: Long)
```

Comment:
`createdNanoTime` or similar pertaining to time heartbeat event was created makes more sense to me and is more standardized IMO. Unless you are modifying the Heartbeat.apply(..) right before it goes out the door which surely we are not. And the messaging sub-system is not modifying when it goes out the door either.